ui_framework/components/form/_color_picker.scss

```diff
@@ -0,0 +1,29 @@
+.kuiColorSwatch {
+  width: 19px;
+  height: 19px;
```

Comment:
Define these are a local var. Should probably be 20px based on other sizing stuff in k5 currently.